vcd/test-templates/misc3.tf

```diff
@@ -0,0 +1,45 @@
+# Remove the first '#' from the next two lines to enable options for terraform executable 
```

Comment:
This sample file is specific to LB. It's probably better to call it load-balancer1.tf

Reply:
Agree. Renamed